### PR TITLE
Missing Language Pack. Update message yii   

### PR DESCRIPTION
**Pull request by liulipeng**
| Q | A |
| --- | --- |
| Is bugfix? | yes |
| New feature? | no |
| Breaks BC? | no |
| Tests pass? | yes |
| Fixed issues | comma-separated list of tickets # fixed by the PR, if any |

### CODE REVIEW

**Comment by cebe** [comment]
Thank you for these translations. However, there are a lot more translations missing in these two languages:

```php
    'Only files with these MIME types are allowed: {mimeTypes}.' => '',
    'Powered by {yii}' => '',
    'The file "{file}" is too big. Its size cannot exceed {formattedLimit}.' => '',
    'The file "{file}" is too small. Its size cannot be smaller than {formattedLimit}.' => '',
    'The requested view "{name}" was not found.' => '',
    'Unknown alias: -{name}' => '',
    'Yii Framework' => '',
    '{attribute} contains wrong subnet mask.' => '',
    '{attribute} is not in the allowed range.' => '',
    '{attribute} must be a valid IP address.' => '',
    '{attribute} must be an IP address with specified subnet.' => '',
    '{attribute} must not be a subnet.' => '',
    '{attribute} must not be an IPv4 address.' => '',
    '{attribute} must not be an IPv6 address.' => '',
    '{delta, plural, =1{1 day} other{# days}}' => '',
    '{delta, plural, =1{1 hour} other{# hours}}' => '',
    '{delta, plural, =1{1 minute} other{# minutes}}' => '',
    '{delta, plural, =1{1 month} other{# months}}' => '',
    '{delta, plural, =1{1 second} other{# seconds}}' => '',
    '{delta, plural, =1{1 year} other{# years}}' => '',
    '{nFormatted} B' => '',
    '{nFormatted} GB' => '',
    '{nFormatted} GiB' => '',
    '{nFormatted} KB' => '',
    '{nFormatted} KiB' => '',
    '{nFormatted} MB' => '',
    '{nFormatted} MiB' => '',
    '{nFormatted} PB' => '',
    '{nFormatted} PiB' => '',
    '{nFormatted} TB' => '',
    '{nFormatted} TiB' => '',
    '{nFormatted} {n, plural, =1{byte} other{bytes}}' => '',
    '{nFormatted} {n, plural, =1{gibibyte} other{gibibytes}}' => '',
    '{nFormatted} {n, plural, =1{gigabyte} other{gigabytes}}' => '',
    '{nFormatted} {n, plural, =1{kibibyte} other{kibibytes}}' => '',
    '{nFormatted} {n, plural, =1{kilobyte} other{kilobytes}}' => '',
    '{nFormatted} {n, plural, =1{mebibyte} other{mebibytes}}' => '',
    '{nFormatted} {n, plural, =1{megabyte} other{megabytes}}' => '',
    '{nFormatted} {n, plural, =1{pebibyte} other{pebibytes}}' => '',
    '{nFormatted} {n, plural, =1{petabyte} other{petabytes}}' => '',
    '{nFormatted} {n, plural, =1{tebibyte} other{tebibytes}}' => '',
    '{nFormatted} {n, plural, =1{terabyte} other{terabytes}}' => '',
```

could you provide translations for these too?

**Comment by liulipeng** [comment]
- zh-CN/yii.php
```
    'Only files with these MIME types are allowed: {mimeTypes}.' => '只允许这些MIME类型的文件: {mimeTypes}。',
    'Powered by {yii}' => '技术支持 {yii}',
    'The file "{file}" is too big. Its size cannot exceed {formattedLimit}.' => '文件"{file}"太大了。它的大小不能超过{formattedLimit}。',
    'The file "{file}" is too small. Its size cannot be smaller than {formattedLimit}.' => '文件"{file}"太小了。它的大小不能超过{formattedLimit}。',
    'The requested view "{name}" was not found.' => '所请求的视图不存在"{name}"。',
    'Unknown alias: -{name}' => '未知的别名: -{name}',
    'Yii Framework' => 'Yii 框架',
    '{attribute} contains wrong subnet mask.' => '{attribute} 属性包含错误的子网掩码。',
    '{attribute} is not in the allowed range.' => '{attribute} 不在允许的范围内。',
    '{attribute} must be a valid IP address.' => '{attribute} 必须是一个有效的IP地址。',
    '{attribute} must be an IP address with specified subnet.' => '{attribute} 必须指定一个IP地址和子网。',
    '{attribute} must not be a subnet.' => '{attribute} 必须不是一个子网。',
    '{attribute} must not be an IPv4 address.' => '{attribute} 必须不是一个IPv4地址。',
    '{attribute} must not be an IPv6 address.' => '{attribute} 必须不是一个IPv6地址。',
    '{delta, plural, =1{1 day} other{# days}}' => '{delta} 天',
    '{delta, plural, =1{1 hour} other{# hours}}' => '{delta} 小时',
    '{delta, plural, =1{1 minute} other{# minutes}}' => '{delta} 分钟',
    '{delta, plural, =1{1 month} other{# months}}' => '{delta} 月',
    '{delta, plural, =1{1 second} other{# seconds}}' => '{delta} 秒',
    '{delta, plural, =1{1 year} other{# years}}' => '{delta} 年',
    '{nFormatted} B' => '{nFormatted} 字节',
    '{nFormatted} GB' => '{nFormatted} GB',
    '{nFormatted} GiB' => '{nFormatted} GiB',
    '{nFormatted} KB' => '{nFormatted} KB',
    '{nFormatted} KiB' => '{nFormatted} KiB',
    '{nFormatted} MB' => '{nFormatted} MB',
    '{nFormatted} MiB' => '{nFormatted} MiB',
    '{nFormatted} PB' => '{nFormatted} PB',
    '{nFormatted} PiB' => '{nFormatted} PiB',
    '{nFormatted} TB' => '{nFormatted} TB',
    '{nFormatted} TiB' => '{nFormatted} TiB',
    '{nFormatted} {n, plural, =1{byte} other{bytes}}' => '{nFormatted} Byte',
    '{nFormatted} {n, plural, =1{gibibyte} other{gibibytes}}' => '{nFormatted} 字节',
    '{nFormatted} {n, plural, =1{gigabyte} other{gigabytes}}' => '{nFormatted} Gigabyte',
    '{nFormatted} {n, plural, =1{kibibyte} other{kibibytes}}' => '{nFormatted} KibiByte',
    '{nFormatted} {n, plural, =1{kilobyte} other{kilobytes}}' => '{nFormatted} Kilobyte',
    '{nFormatted} {n, plural, =1{mebibyte} other{mebibytes}}' => '{nFormatted} MebiByte',
    '{nFormatted} {n, plural, =1{megabyte} other{megabytes}}' => '{nFormatted} Megabyte',
    '{nFormatted} {n, plural, =1{pebibyte} other{pebibytes}}' => '{nFormatted} PebiByte',
    '{nFormatted} {n, plural, =1{petabyte} other{petabytes}}' => '{nFormatted} Petabyte',
    '{nFormatted} {n, plural, =1{tebibyte} other{tebibytes}}' => '{nFormatted} TebiByte',
    '{nFormatted} {n, plural, =1{terabyte} other{terabytes}}' => '{nFormatted} Terabyte',
```
- zh-TW/yii.php
```
    'Only files with these MIME types are allowed: {mimeTypes}.' => '只允許這些MIME類型的文件: {mimeTypes}。',
    'Powered by {yii}' => '技术支持 {yii}',
    'The file "{file}" is too big. Its size cannot exceed {formattedLimit}.' => '檔案"{file}"太大了。它的大小不能超過{formattedLimit}。',
    'The file "{file}" is too small. Its size cannot be smaller than {formattedLimit}.' => '文件"{file}"太小了。它的大小不能超過{formattedLimit}。',
    'The requested view "{name}" was not found.' => '所請求的視圖不存在"{name}"。',
    'Unknown alias: -{name}' => '未知的別名: -{name}',
    'Yii Framework' => 'Yii 框架',
    '{attribute} contains wrong subnet mask.' => '{attribute} 屬性包含錯誤的子網掩碼。',
    '{attribute} is not in the allowed range.' => '{attribute} 不在允許的範圍內。',
    '{attribute} must be a valid IP address.' => '{attribute} 必須是一個有效的IP地址。',
    '{attribute} must be an IP address with specified subnet.' => '{attribute} 必須指定一個IP地址和子網。',
    '{attribute} must not be a subnet.' => '{attribute} 必須不是一個子網。',
    '{attribute} must not be an IPv4 address.' => '{attribute} 必須不是一個IPv4地址。',
    '{attribute} must not be an IPv6 address.' => '{attribute} 必須不是一個IPv6地址。',
    '{delta, plural, =1{1 day} other{# days}}' => '{delta} 天',
    '{delta, plural, =1{1 hour} other{# hours}}' => '{delta} 小時',
    '{delta, plural, =1{1 minute} other{# minutes}}' => '{delta} 分鐘',
    '{delta, plural, =1{1 month} other{# months}}' => '{delta} 月',
    '{delta, plural, =1{1 second} other{# seconds}}' => '{delta} 秒',
    '{delta, plural, =1{1 year} other{# years}}' => '{delta} 年',
    '{nFormatted} B' => '{nFormatted} 字節',
    '{nFormatted} GB' => '{nFormatted} GB',
    '{nFormatted} GiB' => '{nFormatted} GiB',
    '{nFormatted} KB' => '{nFormatted} KB',
    '{nFormatted} KiB' => '{nFormatted} KiB',
    '{nFormatted} MB' => '{nFormatted} MB',
    '{nFormatted} MiB' => '{nFormatted} MiB',
    '{nFormatted} PB' => '{nFormatted} PB',
    '{nFormatted} PiB' => '{nFormatted} PiB',
    '{nFormatted} TB' => '{nFormatted} TB',
    '{nFormatted} TiB' => '{nFormatted} TiB',
    '{nFormatted} {n, plural, =1{byte} other{bytes}}' => '{nFormatted} Byte',
    '{nFormatted} {n, plural, =1{gibibyte} other{gibibytes}}' => '{nFormatted} 字節',
    '{nFormatted} {n, plural, =1{gigabyte} other{gigabytes}}' => '{nFormatted} Gigabyte',
    '{nFormatted} {n, plural, =1{kibibyte} other{kibibytes}}' => '{nFormatted} KibiByte',
    '{nFormatted} {n, plural, =1{kilobyte} other{kilobytes}}' => '{nFormatted} Kilobyte',
    '{nFormatted} {n, plural, =1{mebibyte} other{mebibytes}}' => '{nFormatted} MebiByte',
    '{nFormatted} {n, plural, =1{megabyte} other{megabytes}}' => '{nFormatted} Megabyte',
    '{nFormatted} {n, plural, =1{pebibyte} other{pebibytes}}' => '{nFormatted} PebiByte',
    '{nFormatted} {n, plural, =1{petabyte} other{petabytes}}' => '{nFormatted} Petabyte',
    '{nFormatted} {n, plural, =1{tebibyte} other{tebibytes}}' => '{nFormatted} TebiByte',
    '{nFormatted} {n, plural, =1{terabyte} other{terabytes}}' => '{nFormatted} Terabyte',
```

**Comment by samdark** [comment]
@liulipeng can you add it to the pull request branch? Commit there and it would appear in a pull request automatically.

**Comment by liulipeng** [comment]
Updated language packs zh-CN and ZH-TW

**Comment by liulipeng** [comment]
Hello, it has been added to the pull request branch


刘李鹏
手机 13366941391
公司 北京纽斯洛网络科技有限公司
QQ           619119597
微信ID     liulipeng01      
Email  <mailto:liu.lipeng@newsnow.com.cn>liu.lipeng@icloud.com <mailto:liu.lipeng@newsnow.com.cn>

File: framework/messages/zh-CN/yii.php [hunk 1]
```diff
@@ -85,11 +98,39 @@
...
+    '{nFormatted} KB' => '{nFormatted} KB',
+    '{nFormatted} KiB' => '{nFormatted} KiB',
+    '{nFormatted} MB' => '{nFormatted} MB',
+    '{nFormatted} MiB' => '{nFormatted} MiB',
+    '{nFormatted} PB' => '{nFormatted} PB',
+    '{nFormatted} PiB' => '{nFormatted} PiB',
+    '{nFormatted} TB' => '{nFormatted} TB',
+    '{nFormatted} TiB' => '{nFormatted} TiB',
+    '{nFormatted} {n, plural, =1{byte} other{bytes}}' => '{nFormatted} Byte',
+    '{nFormatted} {n, plural, =1{gibibyte} other{gibibytes}}' => '{nFormatted} 字节',
```
Comment (samdark) [inline framework/messages/zh-CN/yii.php]: gibibyte is not the same as byte

File: framework/messages/zh-CN/yii.php [hunk 2]
```diff
@@ -85,11 +98,39 @@
...
+    '{nFormatted} {n, plural, =1{gigabyte} other{gigabytes}}' => '{nFormatted} Gigabyte',
```
Comment (samdark) [inline framework/messages/zh-CN/yii.php]: Why ↑ is translated while ↓ aren't?

File: framework/messages/zh-TW/yii.php [hunk 3]
```diff
@@ -84,11 +97,39 @@
...
+    '{nFormatted} KB' => '{nFormatted} KB',
+    '{nFormatted} KiB' => '{nFormatted} KiB',
+    '{nFormatted} MB' => '{nFormatted} MB',
+    '{nFormatted} MiB' => '{nFormatted} MiB',
+    '{nFormatted} PB' => '{nFormatted} PB',
+    '{nFormatted} PiB' => '{nFormatted} PiB',
+    '{nFormatted} TB' => '{nFormatted} TB',
+    '{nFormatted} TiB' => '{nFormatted} TiB',
+    '{nFormatted} {n, plural, =1{byte} other{bytes}}' => '{nFormatted} Byte',
+    '{nFormatted} {n, plural, =1{gibibyte} other{gibibytes}}' => '{nFormatted} 字節',
```
Comment (samdark) [inline framework/messages/zh-TW/yii.php]: See https://github.com/yiisoft/yii2/pull/11769/files/7a9955e55050135feaf26c7af045d28ed398fbe8..fa05614cc391a18fb0839905c58503a9fdd411fb#r67613159

**Comment by liulipeng** [comment]
Thanks for asking questions, is my problem, I'm sorry

Last updated incorrect translation and language no translation

**Comment by samdark** [comment]
Looks good to me now. @cebe ?

**Comment by cebe** [comment]
Looks good, merged it. Thank you @liulipeng!

**Pull request closed without merging**